dsn.go

```diff
@@ -14,7 +14,7 @@ type Config struct {
 	Addr       string
 	DBName     string
 	Auth       string
-	Batch      int
+	Batch      uint32
```

Comment:
Many thanks for this PR! In fact, I still insist on using `int` `Batch`, for:
1. `Batch` won't touch the upper bound of `int`.
2. It will be incompatible for other projects which depend on this `gohive`.

Could you provide more explanations？

Reply:
Done